src/js/App/Header/ContextSwitcher.js

```diff
@@ -32,6 +39,8 @@ const ContextSwitcher = ({ user, className }) => {
     localStorage.removeItem(REQUESTS_DATA);
     setSelectedAccountNumber(target_account);
     Cookies.set(CROSS_ACCESS_ACCOUNT_NUMBER, target_account);
+    Cookies.set(TARGET_ORG, target_account);
```

Comment:
Hey @arivepr! This will need to be the org id value, not the account number. The idea is that RBAC will need both the account number in the `cross_access_account_number` cookie, and the org id in the `cross_access_org_id` cookie.

Reply:
Hey Keith @coderbydesign , thanks for the clarification!

Did not realize they were two seperate cookies.
If you don't mind, does that look good for the crossid? Or is it a naming issue, given that target_org === cross_id? and then add the missing cookie/constant for cross-act-num

Reply:
Not sure I follow completely, but since these two cookies should be named similarly I would think. So where you have `CROSS_ACCESS_ACCOUNT_NUMBER` currently, I'd think it would make sense to have `CROSS_ACCESS_ORG_ID` instead of `TARGET_ORG`, but you'd supply it the value `target_org`, from the actual cross account request object?

The actual cookie name of `cross_access_org_id` is perfect, though! That's what we're anticipating be sent.